src/oidcendpoint/endpoint.py

```diff
@@ -197,6 +198,7 @@ def __init__(self, endpoint_context, **kwargs):
         # This is for matching against aud in JWTs
         # By default the endpoint's endpoint URL is an allowed target
         self.allowed_targets = [self.name]
+        self.client_verification_method = []
```

Comment:
This is unused

Reply:
we could execute autopep8 and autoflake on this branch

Reply:
It's needed by OIDC federation.